stack_examples/aws_ssm_package_installer/main.tf

```diff
@@ -0,0 +1,141 @@
+resource "turbot_policy_setting" "ssmStackSource" {
+  resource = var.region_resource
+  type     = "tmod:@turbot/aws-ssm#/policy/types/ssmStackSource"
+  value    = <<POLICY_VALUE
+    resource "aws_ssm_document" "turbot_linux_package_installer_public_network_document" {
+      name            = "Turbot-LinuxPackageInstaller-PublicNetwork"
+      document_type   = "Command"
+      document_format = "YAML"
+      content         = <<DOC
+        schemaVersion: "2.2"
+        description: Installs package on Linux Instance from OS package source repository
+        parameters:
+          package:
+            type: String
+            description: (Required) The package to be installed
+            minChars: 1
+            default: "${var.default_package}"
+        mainSteps:
+          - action: "aws:runShellScript"
+            name: InstallPackage
+            inputs:
+              runCommand:
+                - |
+                  #!/bin/bash
+                  set -e
+
+                  OS="$(hostnamectl | sed -n -e 's/^.*Operating System: //p')"
+                  echo "OS is: $OS"
+
+                  if [[ $OS =~ "Ubuntu" ]]; then
+                    apt-get update
+                    apt-get install -y {{ package }}
+                  fi
+    DOC
+    }
+
+    resource "aws_ssm_document" "turbot_linux_s3_package_installer_document" {
```

Comment:
Maybe a useful comment for each resource that you are applying to the infrastructure and its purpose may be helpful